http_check/datadog_checks/http_check/http_check.py

```diff
@@ -131,7 +131,7 @@ def get_ca_certs_path():
     """
     Get a path to the trusted certificates of the system
     """
-    ca_certs = ['/opt/datadog-agent/embedded/ssl/certs/cacert.pem']
+    ca_certs = [os.path.normpath(os.path.join(os.path.dirname(__file__), '../../../..', 'ssl/certs', 'cacert.pem'))]
```

Comment:
Can we add a comment to explain where `../../../..` come from and where it will bring us ?